ecommerce-demo/workspaces/childDemo_electronics/workspaceSettings.json

```diff
@@ -0,0 +1,204 @@
+{
+  "colorPalette": {
+    "colorPaletteId": "demo_contrast",
```

Comment:
Should we change `"demo_contrast"` → `"demo_colorPalette_contrast"` to be consistent with the naming for the theme id (at line 88), with has a value of `"demo_theme_xxx"`.